template/web/src/components/contract-alert/ContractAlert.tsx

```diff
@@ -0,0 +1,58 @@
+import { ExclamationTriangleIcon } from '@radix-ui/react-icons';
+import { Abi, parseEther } from 'viem';
+
```

Comment:
More a nit,
I am reflecting if we should standardize around component-name on the folder or use the ComponentName as folder as well.

Reply:
I like when folder name is used like `group-name` and keeping things flat whenever possible with this file path `@/component/ContractAlert.tsx`.

This component could be grouped with `@/hooks/contracts` and `@/hooks/useUserCanAffrod`.

Reply:
To be specific, this file has a redundant folder. We are gonna to be moving fast and reducing things to do sounds good to me.

Reply:
I like `@/component/ContractAlert.tsx` better as well, and have `test` folder for tests.